src/sc_kpm/utils/action_utils.py

```diff
@@ -32,11 +38,26 @@ def check_action_class(action_class: Union[ScAddr, Idtf], action_node: ScAddr) -
     return len(search_results) > 0
 
 
-def get_action_answer(action: ScAddr) -> ScAddr:
-    templ = ScTemplate()
+def get_arguments(action_node: ScAddr, count: int) -> List[ScAddr]:
```

Comment:
rename to "get_action_arguments", it sounds more completely for me

Reply:
Done